modules/persistent-storage-csi-migration-vsphere.adoc

```diff
@@ -6,35 +6,61 @@
 [id="persistent-storage-csi-migration-sc-vsphere_{context}"]
 = vSphere automatic migration
 
+[IMPORTANT]
+====
+Migration includes significant consequences:
+
+* Enabling migration cannot be undone.
+
+* Migration can take a while to complete depending on how many nodes are on the cluster.
+====
+
 == New installations of {product-title}
 For new installations of {product-title} 4.13, or later, automatic migration is enabled by default.
 
-[id="upgrading-openshift_{context}"]
-== Upgrading {product-title}
-When upgrading from {product-title} 4.12, or earlier, to 4.13, automatic CSI migration for vSphere only occurs if you opt in.
+== Updating from {product-title} 4.13 to 4.14
+If you are using vSphere in-tree persistent volumes (PVs) and want to update from {product-title} 4.13 to 4.14, first update vSphere vCenter and ESXI host to 7.0 Update 3L or 8.0 Update 2, otherwise the {product-title} update is blocked. After updating vSphere, your {product-title} update can occur and automatic migration is enabled by default.
```

Comment:
unless CSI migration is already enabled
cc @gnufied

Reply:
Correct me if I'm wrong, but if you have enabled migration, then you won't have in-tree PVs because they have been migrated to CSI, so the first phrase, I think, covers this. @gnufied @gcharot

Reply:
The in-tree PVs still appear at in-tree but yes fair let's not complexify the wording.